task_1/2.js

```diff
@@ -0,0 +1,2 @@
+// Написать функцию для подсчета количества букв в слове
+charsCounter = (word) => { return word.length }
```

Comment:
Для такой простой стрелочной функции можно уменьшить количество скобок и сократить написание оператора `return`

```suggestion
charsCounter = word => word.length
```